cypress/integration/Editor/versions_spec.js

```diff
@@ -33,17 +30,16 @@ describe('Workflow features', () => {
     cy.get('[data-testid=saveLearningResourceButtonWrapper] button')
       .first()
       .click();
+    cy.apiwait('@patchUserData');
```

Comment:
At denne mangla såg ut til å rote det til.

Reply:
Du har nok rett, den og testen under overflower til `film_spec` når alle tester kjøres sammen, som gjør at `film_spec` failer uten om at det er dens skyld. Har en fix på det, pusher snart

Reply:
Close, but no cigar!

Reply:
Trur ikkje du skulle reverta version_spec.

Reply:
Det triggrer meg at jeg ikke får de samme feilene lokalt, alle testene passer lokalt

Reply:
Kun to igjen no...